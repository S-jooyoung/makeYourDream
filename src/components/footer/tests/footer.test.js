import React from "react";

import Footer from "../footer";
import renderer from "react-test-renderer";

describe("Footer", () => {
  let FooterComponent;

  beforeEach(() => {
    FooterComponent = <Footer />;
  });

  it("render", () => {
    const component = renderer.create(FooterComponent).toJSON();
    expect(component).toMatchSnapshot();
  });
});
