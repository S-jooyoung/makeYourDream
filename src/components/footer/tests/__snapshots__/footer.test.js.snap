// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Footer render 1`] = `
<footer
  className="footer"
>
  <h1
    className="content"
  >
    © 2022 All Rights Reserved, Jooyoung®
  </h1>
</footer>
`;
